utils/scapp.h

```diff
@@ -895,6 +915,9 @@ scApp::processOption(argparser& parser, int opt)
         options.ktx2 = 1;
         break;
       case 1016:
+        options.normalize = true;
```

Comment:
Shouldn't we set `options.normalize` too if -`-normal_mode` is specified? Failure to normalize means the 2-component normal map isn't usable as the correct 3-component normal is not recoverable. On the other hand, if the input is already normalized setting it will consume a lot of unnecessary cpu cycles. What do you think?

Reply:
Good point. I think I prefer it to be explicit. If we couple the features then its not possible to use `--normal_mode` without it if someone choses to trust their normal map. I have noticed by playing with NVIDIA texture tool that normalising can result in loss of quality or I should say saturation. There might be cases where people are happy with the slight inaccurate result but prefer the saturated output (Very hard to explain this in words unless you see the normal maps I am talking about).

On the other hand its not always clear why the generated 2-component normal map doesn't look right if you don't know the specifics of "non-normalised nomal map to 2-component" coversion doesn't work and you need to do that. So perhaps a warning would be appropriate if that happens but it might be exessive as well.